### PR TITLE
Add ppc64le support

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by 6543** [comment]
@isuruf thanks for the pull - we solved it at the plugin side ;)

https://github.com/woodpecker-ci/plugin-git/releases/tag/v1.1.0

closed by https://github.com/woodpecker-ci/plugin-git/pull/8

**Comment by isuruf** [comment]
Thanks @6543. What about the changes in Makefile?

**Comment by 6543** [comment]
@isuruf to what changes you refer to?

File: Makefile [hunk 1]
```diff
@@ -86,32 +86,36 @@ release-server:
 
 release-agent:
 	# compile
-	GOOS=linux   GOARCH=amd64 CGO_ENABLED=0 go build -ldflags '${LDFLAGS}' -o dist/agent/linux_amd64/woodpecker-agent   github.com/woodpecker-ci/woodpecker/cmd/agent
-	GOOS=linux   GOARCH=arm64 CGO_ENABLED=0 go build -ldflags '${LDFLAGS}' -o dist/agent/linux_arm64/woodpecker-agent   github.com/woodpecker-ci/woodpecker/cmd/agent
-	GOOS=linux   GOARCH=arm   CGO_ENABLED=0 go build -ldflags '${LDFLAGS}' -o dist/agent/linux_arm/woodpecker-agent     github.com/woodpecker-ci/woodpecker/cmd/agent
-	GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build -ldflags '${LDFLAGS}' -o dist/agent/windows_amd64/woodpecker-agent github.com/woodpecker-ci/woodpecker/cmd/agent
-	GOOS=darwin  GOARCH=amd64 CGO_ENABLED=0 go build -ldflags '${LDFLAGS}' -o dist/agent/darwin_amd64/woodpecker-agent  github.com/woodpecker-ci/woodpecker/cmd/agent
-	GOOS=darwin  GOARCH=arm64 CGO_ENABLED=0 go build -ldflags '${LDFLAGS}' -o dist/agent/darwin_arm64/woodpecker-agent  github.com/woodpecker-ci/woodpecker/cmd/agent
+	GOOS=linux   GOARCH=amd64   CGO_ENABLED=0 go build -ldflags '${LDFLAGS}' -o dist/agent/linux_amd64/woodpecker-agent   github.com/woodpecker-ci/woodpecker/cmd/agent
```
Comment (isuruf) [inline Makefile:89]: @6543, the changes here

**Comment by 6543** [comment]
thats comming from https://github.com/woodpecker-ci/woodpecker/pull/408 - but will be changed since we like to address https://github.com/woodpecker-ci/woodpecker/issues/187 and similar to https://github.com/woodpecker-ci/plugin-git/pull/8

**Pull request closed without merging**